leaf-ui/css/analysis_style.css

```diff
@@ -1,18 +1,22 @@
 .modal {
     position: fixed; /* Stay in place */
-    z-index: 1; /* Sit on top */
```

Comment:
Just wanted to make sure but why did you take this out?

Reply:
I edited the wrong class; I was looking for modal-content instead.